x/qgb/keeper/keeper_valset.go

```diff
@@ -130,7 +130,7 @@ func normalizeValidatorPower(rawPower uint64, totalValidatorPower cosmosmath.Int
 
 // GetLastValsetBeforeNonce returns the previous valset before the provided `nonce`.
 // the `nonce` can be a valset, but this method will return the valset before it.
-// If the provided nonce is 1. It will return an error. Because, there is no valset before nonce 1.
+// If the provided nonce is 1, it will return an error, because, there is no valset before nonce 1.
 func (k Keeper) GetLastValsetBeforeNonce(ctx sdk.Context, nonce uint64) (*types.Valset, error) {
```

Comment:
Shouldn't the second argument be `*QueryLastValsetRequestBeforeNonceRequest`? Or where does the actual query get called to

Reply:
That's the keeper. It's called in the querier here: https://github.com/celestiaorg/celestia-app/blob/4f9deb07b72e34f5eeeda87ef2a70f45f05c1c3f/x/qgb/keeper/query_valset.go#L13-L16

That change is just a docs change that Sanaz spotted and I fixed it in this PR (instead of opening a new one)